library_generation/generate_pr_description.py

```diff
@@ -121,7 +131,15 @@ def __get_commit_messages(
     shutil.rmtree(tmp_dir, ignore_errors=True)
     os.mkdir(tmp_dir)
     repo = Repo.clone_from(repo_url, tmp_dir)
-    commit = repo.commit(latest_commit)
+    commit = repo.commit(current_commit)
+    current_commit_time = __get_commit_timestamp(commit)
+    baseline_commit_time = __get_commit_timestamp(repo.commit(baseline_commit))
+    if current_commit_time < baseline_commit_time:
```

Comment:
Like this validation!  I think we should add `=` though.
```suggestion
    if current_commit_time <= baseline_commit_time:
```

Reply:
Done.

Added unit tests to verify the behavior.